src/include/nodes/memnodes.h

```diff
@@ -17,6 +17,37 @@
 #define MEMNODES_H
 
 #include "nodes/nodes.h"
+#include "utils/hsearch.h"
+#include "access/hash.h"
+
+#define CHUNKS_TABLE_SIZE 1024
+
+typedef struct
+{
+	uint64_t count;
+	uint64_t bytes;
+} MemoryContextChunkStat;
```

Comment:
If we back to a modular coding model, please look if this structures can be moved to a .c file along with all functions which use them.